src/components/WalletStatementModal/index.tsx

```diff
@@ -19,19 +22,42 @@ function WalletStatementModal({statementPageURL, session}: WalletStatementProps)
      * Handles in-app navigation for iframe links
      */
     const navigate = (event: MessageEvent<WalletStatementMessage>) => {
-        if (!event.data?.type || (event.data.type !== CONST.WALLET.WEB_MESSAGE_TYPE.STATEMENT && event.data.type !== CONST.WALLET.WEB_MESSAGE_TYPE.CONCIERGE)) {
+        const {data} = event;
+        const {type, url} = data || {};
+        if (!type || (type !== CONST.WALLET.WEB_MESSAGE_TYPE.STATEMENT && type !== CONST.WALLET.WEB_MESSAGE_TYPE.CONCIERGE)) {
             return;
         }
 
-        if (event.data.type === CONST.WALLET.WEB_MESSAGE_TYPE.CONCIERGE) {
-            Report.navigateToConciergeChat(true);
+        if (type === CONST.WALLET.WEB_MESSAGE_TYPE.CONCIERGE) {
+            navigateToConciergeChat(true);
```

Comment:
```suggestion
            navigateToConciergeChat();
```
To avoid issue https://github.com/Expensify/App/pull/63017

Reply:
Yes, I think so.